west.yml

```diff
@@ -126,7 +126,7 @@ manifest:
       groups:
         - hal
     - name: hal_silabs
-      revision: 249c08f16f5ee9663c8b225b68faf8ec54a21e8e
+      revision: 785f4ed3aadb57d1b694be48c6f06fd5617df609
```

Comment:
Wrong revision? I think you want to point to your PR in the `hal_silabs` repository with `pull/<pr_num>/head` here.